spec/commands/inject_spec.rb

```diff
@@ -52,6 +52,31 @@
     end
   end
 
+  context "with source option" do
+    it "add gem with source option in gemfile" do
+      bundle "inject 'foo' '>0' --source file://#{gem_repo1}"
+      gemfile = bundled_app("Gemfile").read
+      str = "gem 'foo', '> 0', :source => 'file://#{gem_repo1}'"
+      expect(gemfile).to include str
+    end
+  end
+
+  context "with group option" do
+    it "add gem with group option in gemfile" do
+      bundle "inject 'rack-obama' '>0' --group=development"
+      gemfile = bundled_app("Gemfile").read
+      str = "gem 'rack-obama', '> 0', :group => [:development]"
+      expect(gemfile).to include str
+    end
+
+    it "add gem with multiple groups in gemfile" do
+      bundle "inject 'rack-obama' '>0' --group=development,test"
+      gemfile = bundled_app("Gemfile").read
+      str = "gem 'rack-obama', '> 0', :group => [:development, :test]"
```

Comment:
@segiddins , I saw example here : http://bundler.io/v1.3/groups.html . In gemfile it must be `:group => [:development, :test]` right ? I haven't see `groups ` in gemfile.  

To install all dependencies, except those in specified groups
we do something : 
```
$ bundle install --without test development
```
will it work when we write `groups` ?